github/ipfs.yml

```diff
@@ -5340,11 +5311,6 @@ repositories:
             - go-test
             - sharness-test
           strict: true
-    collaborators:
-      admin:
-        - lidel
-      push:
-        - dennis-tra
```

Comment:
@dennis-tra I'm removing your permissions here and there will be a follow up PR to add you to the IPFS org and from there you can get added to any teams you need to be.

Reply:
Sounds good, thanks for the heads-up! 👍